Dockerfile

```diff
@@ -7,6 +7,6 @@ RUN pip install -r /uatframework/requirements.txt
 ADD features /uatframework/features
 ADD steps /uatframework/steps
 ADD environment.py /uatframework/environment.py
-ADD config/central_ci_dynamic_hosts.py /uatframework/config/central_ci_dynamic_hosts.py
+LABEL RUN="docker run -it -v $HOME/.ssh:/root/.ssh -v /path/to/UATFramework/resources.json:/uatframework/resources.json -v /path/to/UATFramework/config:/uatframework/config aweiteka/uatframework"
```

Comment:
Mounting in the `$HOME/.ssh` directory is kind of a hack.  We should have a better story for using SSH keys with provisioned resources.